cli/installation/index.md

```diff
@@ -126,22 +134,27 @@ scoop update keboola/keboola-cli
 
 ### Download
 
-Use an [msi installer](https://cli-dist.keboola.com/?prefix=msi/) or a [precompiled binary](https://cli-dist.keboola.com/?prefix=zip/). 
-You can also install from the source.
-
-## Installing from Source
+Use an [msi installer](https://cli-dist.keboola.com/?prefix=msi/) or a [precompiled binary](https://cli-dist.keboola.com/?prefix=zip/).
+## Build From Source
 
 1. Install the [Go environment](https://golang.org/doc/install) (if you haven't done so already).
-2. Clone the Git source (or if you don't yet have a version of Git installed,
-   [download and extract it](https://github.com/keboola/keboola-as-code/archive/refs/heads/main.zip)):
-
-        git clone https://github.com/keboola/keboola-as-code
-
-3. Run the installation script:
+2. Clone the source from GitHub:
+```
+git clone https://github.com/keboola/keboola-as-code
+cd keboola-as-code
+```
 
-        make build-local
+3. Run the build:  
+On Linux or macOS:
+```
+go build -o target/kbc ./cmd/kbc/main.go
+```
+On Windows:
+```
+go build -o target/kbc.exe ./cmd/kbc/main.go
+```
```

Comment:
`make build-local` vyzaduje goreleaser ... tak som to upravil.
Na Go je krasne, ze nepotrebujes obycajne ziadne dalsie zavislosti.